tests/scripts/all.sh

```diff
@@ -469,101 +467,62 @@ tests/scripts/doxygen.sh
 
 
 ################################################################
-#### Build and test many configurations and targets
+#### 1 Basic tests
 ################################################################
 
-if [ $RUN_ARMCC -ne 0 ] && [ $YOTTA -ne 0 ]; then
-    # Note - use of yotta is deprecated, and yotta also requires armcc to be on the
-    # path, and uses whatever version of armcc it finds there.
-    msg "build: create and build yotta module" # ~ 30s
-    cleanup
-    record_status tests/scripts/yotta-build.sh
-fi
+# 1.1 Basic tests: default config
+#################################
 
-msg "build: cmake, gcc, ASan" # ~ 1 min 50s
+msg "build: cmake, gcc, ASan" # ~ 1 min
 cleanup
 CC=gcc cmake -D CMAKE_BUILD_TYPE:String=Asan .
 make
 
-msg "test: main suites (inc. selftests) (ASan build)" # ~ 50s
+msg "test: main suites (inc. selftests) (ASan build)" # < 10s
 make test
 
-msg "test: ssl-opt.sh (ASan build)" # ~ 1 min
+msg "test: ssl-opt.sh (ASan build)" # ~ 2 min
 if_build_succeeded tests/ssl-opt.sh
 
-msg "test/build: ref-configs (ASan build)" # ~ 6 min 20s
-record_status tests/scripts/test-ref-configs.pl
-
-msg "build: with ASan (rebuild after ref-configs)" # ~ 1 min
-make
-
 msg "test: compat.sh (ASan build)" # ~ 6 min
 if_build_succeeded tests/compat.sh
 
-msg "build: Default + SSLv3 (ASan build)" # ~ 6 min
```

Comment:
I can see this test along with the other SSLv3 tests have been removed, but it doesn't seem to have been replaced anywhere. Why not? Am I missing something?

Reply:
Yes is has been replaced, in the "full config" section (where is it can actually be more thoroughly tested, eg including single-DES suites in SSLv3 mode).

Are you reviewing the full diff or individual commits? I know various people have various preferences about that (and the github interface has a number of issues with per-commit reviews unfortunately) but for this particular PR I'd really recommend reviewing commit by commit, as otherwise the big "reorder" commit makes other changes nearly impossible to follow.

This change is https://github.com/ARMmbed/mbedtls/pull/1243/commits/29a6e3432baf2ec1955c4a405e0454adc715a744 where it's easier to see the added line (and hopefully the commit message should help too).